src/shared/components/case-editor/case-edit-form/case-edit-form.html

```diff
@@ -1,15 +1,17 @@
 <ng-container *ngFor="let field of fields">
     <div ccdConditionalShow ccdLabelSubstitutor [caseField]="field" [formGroup]="formGroup" [eventFields]="eventFields">
       <ng-container [ngSwitch]="field | ccdIsReadOnlyAndNotCollection ">
-        <ccd-field-read *ngSwitchCase="true" [caseField]="field" [withLabel]="true"></ccd-field-read>
+        <ccd-field-read *ngSwitchCase="true" [caseField]="field" [eventFields]="eventFields" [withLabel]="true"></ccd-field-read>
```

Comment:
Just initial thought - why are we passing this here? I worked on similar story for making case history a flexible field and Fatih and I agreed we should not be cluttering ccd-field-read with eventFields therefore moved the logic to the BE. Cannot we do that here as well?

Reply:
Can you let me know why do you need eventFields so we can think together how to better design it?